exercises/F-strings-methods/exercise2.js

```diff
@@ -1,3 +1,4 @@
-const name = " Daniel  ";
-
+var message = " Daniel  ";
+var messageLength = message.length;
```

Comment:
It would be good to report the length of message after whitespace has been trimmed off, how can you do that?